src/database/cassandra-handler/init-scripts/16__tests_processors_materialized_view.cql

```diff
@@ -0,0 +1,7 @@
+CREATE MATERIALIZED VIEW tests_processors_id_mapping AS
+
+       SELECT id,processor_id, updated_at FROM tests
+
+       WHERE id IS NOT NULL AND processor_id IS NOT NULL AND updated_at IS NOT NULL
+
+       PRIMARY KEY ((processor_id), id, updated_at)
```

Comment:
do we need updated_at?

Reply:
we need updated_at IS NOT NULL for sure otherwise you would get an error, let me re-check about having it in the primary key

Reply:
yes we must have `updated_at` as a primary key, if I remove it, I get the following error 

`ResponseError: Cannot create Materialized View tests_processors_id_mapping without primary key columns from base tests (updated_at)`